frontend/src/shared/schemas/validation/email.ts

```diff
@@ -0,0 +1,6 @@
+import * as yup from 'yup';
+
+export const email = yup
```

Comment:
i think we should give the validators more descriptive names. maybe something like `emailValidator`?